internal/xds/translator/listener.go

```diff
@@ -39,6 +40,7 @@ func buildXdsTCPListener(name, address string, port uint32) *listenerv3.Listener
 				Filter:     listenerAccessLogFilter,
 			},
 		},
+		PerConnectionBufferLimitBytes: wrapperspb.UInt32(32768),
```

Comment:
would be great if this is a magic number const, with a link to https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto#envoy-v3-api-field-config-listener-v3-listener-per-connection-buffer-limit-bytes

Reply:
Done